sql/moz-fx-data-shared-prod/stripe_derived/plans_v1/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+friendly_name: Stripe Plans
+description: Stripe plans as of UTC midnight
```

Comment:
Might be useful to include the bug number in the metadata section for reference.